pgmpy/base/DAG.py

```diff
@@ -1902,6 +1902,66 @@ def edge_strength(self, data, edges=None):
 
         return strengths
 
+    def validate(self, data, metrics: Optional[tuple[str | Callable]] = None, **kwargs):
```

Comment:
It might also be better to list all the arguments here instead of `kwargs` with some defaults.